org.kde.konsole.json

```diff
@@ -0,0 +1,46 @@
+{
+    "id": "org.kde.konsole",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-22.08",
+    "sdk": "org.kde.Sdk",
+    "command": "konsole",
+    "rename-icon": "utilities-terminal",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--talk-name=org.a11y.Bus",
```

Comment:
This should be removed, see https://github.com/flathub/org.kde.ark/commit/ec438fb43c7ec1f71b0c946f5c54232723a7f8ba

Reply:
Good catch